homeassistant/components/google_assistant/http.py

```diff
@@ -219,6 +236,56 @@ async def async_report_state(self, message, agent_user_id: str):
         }
         await self.async_call_homegraph_api(REPORT_STATE_BASE_URL, data)
 
+    @callback
+    def async_enable_local_sdk(self):
+        """Enable the local SDK."""
+        for key, value in self._store.agent_user_ids.items():
```

Comment:
please pick more descriptive variable names.

Reply:
You are right :)